routes/results.js

```diff
@@ -1,9 +1,24 @@
-const fs = require('fs')
+const dbFunctions = require('../database/db_functions.js')
+const convertFunctions = require('../helpers/convertDis.js')
+module.exports = (req, res) => {
+  const disabilityOption = req.query.disability
+  const serviceOption = convertFunctions.capitalise(req.query.service)
```

Comment:
due to an error on our part we have set up the query string to use "services" not "service" so I will change this here to make sure it works